typedb/api/connection/options.py

```diff
@@ -46,6 +47,69 @@ def cluster() -> "TypeDBClusterOptions":
     def is_cluster(self) -> bool:
         return False
 
+    def get_infer(self) -> Optional[bool]:
```

Comment:
The reason I didn't add these methods in the first place was because accessing the attributes (akin to fields in Java) directly is more "Pythonic".

I would recommend deleting them, but we may want a broader discussion on it.

The main thing missing from the methods (which is still missing in this implementation!) is value validation. But this can be achieved in Python using `@property` and `@batch_size.setter` annotations (which I didn't know about / wasn't comfortable using when I originally wrote `options.py`)

Reply:
We can discuss, but they make it possible to create and set options in one line, rather than spanning multiple lines which I think is still useful!